internal/health/checker/builtin/argocd_test.go

```diff
@@ -458,7 +458,7 @@ func Test_argocdUpdater_getApplicationHealth(t *testing.T) {
 				Namespace: testApp.Namespace,
 				Name:      testApp.Name,
 			},
-			[]string{"fake-version", "fake-commit", "another-fake-commit"},
+			nil,
```

Comment:
This change to the test case demonstrably prevents the cool down period from being enforced, until the corresponding changes to the code under test are introduced.